apps/projects/app/components/Content/Issues.js

```diff
@@ -404,6 +426,39 @@ class Issues extends React.PureComponent {
     })
   }
 
+  getRowRenderer = issuesSorted => ({
+    index,
+    key,
+    parent,
+    style
+  }) => {
+    const issueStyle = {}
+    if(index === 0) {
+      issueStyle.borderRadius = '3px 3px 0 0'
+    }
+    if(index === issuesSorted.length - 1) {
+      issueStyle.borderRadius = '0 0 3px 3px'
```

Comment:
I noticed that the bottom border on the last item is clipped:

<img width="396" alt="Screen Shot 2019-07-16 at 14 33 37" src="https://user-images.githubusercontent.com/221614/61320336-39245380-a7d7-11e9-8402-cd821fe87f3f.png">

Do you think you could look into what causes that? 

Also, I'd rather we did this with CSS, rather than JavaScript. Is there a way to do that?

Reply:
So what I did to solve this was I added border styles to the top and bottom of the list itself.